ballerina/Ballerina.toml

```diff
@@ -2,24 +2,177 @@
 distribution = "2201.9.2"
 org = "ballerinax"
 name = "aws.marketplace.mpe"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
Just to clarify, why not 1.0? :)

Reply:
We have not yet tested these connectors against a live server as we don't have access to market place account with products (as this connector is used to get usage info about marketplace products).